.github/workflows/tag.yml

```diff
@@ -1,13 +1,15 @@
-name: main push autotag
+name: Auto-Tag
 
 on:
-  push:
-    branches:
-      - main
+  workflow_run:
+    workflows: [build]
+    types:
+      - completed
```

Comment:
Ahha I think my question in my previous comment just clicked that the event is in scope because of the specific way this is being triggered. Feel free to disregard